src/lib/utility.js

```diff
@@ -124,6 +124,57 @@ module.exports = (requires) => {
   utilities.filter = (str) => {
     return str.replace('%prefix', config.prefix);
   };
+
+  //for finding the command object for either the keyword or an alias 
+  utilities.getCommand = function(keyword) 
+  { 
+    let commands = info.commands;
+    if(commands[keyword] && typeof commands[keyword].getAction() === 'function') { 
+      return commands[keyword]; 
+    } else { 
+      //didn't find command 
+      for(let index in commands) { 
+        if(commands[index] && typeof commands[index] === 'object'
+           && commands[index].getAlias().indexOf(keyword) > -1) { 
+          return commands[index]; 
+        } 
+      } 
+    } 
+    return null; 
+  };
+
+  utilities.getPermLevel = function(details) {
```

Comment:
I'd suggest making this a promise so you know for sure you'll get what you want.